supernode/daemon/mgr/gc/gc_peer.go

```diff
@@ -46,18 +46,17 @@ func (gcm *Manager) gcPeers(ctx context.Context) {
 			continue
 		}
 
-		if !gcm.gcPeer(ctx, peerID) {
-			logrus.Warnf("gc peers: failed to gc peer peerID(%s): %v", peerID, err)
-			continue
-		}
+		gcm.gcPeer(ctx, peerID)
 		gcPeerCount++
```

Comment:
You can increase the metric here. metric.withlabelvalues().Add(gcPeerCount)

Reply:
@SataQiu  gc_peers and gc_tasks represent the total peers and tasks being gc rather than total times of calling gc functions. So you have to change it here. Use Add(gcPeerCount) instead